lte/gateway/python/magma/mobilityd/tests/rpc_servicer_tests.py

```diff
@@ -58,7 +58,7 @@ def setUp(self):
                                            store)
 
         # Add the servicer
-        self._servicer = MobilityServiceRpcServicer(self._allocator)
+        self._servicer = MobilityServiceRpcServicer(self._allocator, False)
```

Comment:
shouldn't it be False by default? Or you want to make it explicit for the test?

Reply:
Yes I wanted to make it explicit here. 
I was debugging with this on. I think it is useful to have it there to help engineers to see they can  switch back and forth.